doc/source/parameters/cfd/non-linear_solver_control.rst

```diff
@@ -4,6 +4,9 @@ Non-linear Solver
 
 The Navier-Stokes equations (and others) are non-linear equations. The parameters in ``subsection non-linear solver``, whose default values are given in the text block below, control the non-linear solver used within Lethe. Lethe supports different physics (``fluid dynamics``, ``VOF``, ``heat transfer``, ``cahn hilliard`` and ``tracer``) and it is possible to specify non-linear solver parameters for each of them. In the example below, only ``fluid dynamics`` is shown but the same block can be used for other physics.
 
+.. note::
+  The ``VOF algebraic interface reinitialization`` subequation has also its own subsection.
+
```

Comment:
Not sure this needs to be a note... but why not

Reply:
I could move it up; I just was not sure how to fit it before. I could put it before "In the example below [...]".